playbooks/cluster_setup_ceph.yaml

```diff
@@ -18,17 +18,17 @@
   become: true
   gather_facts: yes
   roles:
-    - ceph_expansion_vg
-
+    - role: ceph_expansion_vg
+      when: ansible_distribution | regex_search("Debian")
```

Comment:
No it should work on Yocto too. Furthermore, it runs only when ansible_lvm is defined and lvm_volumes.